client/OPUNetTransportLayer.h

```diff
@@ -39,6 +39,13 @@ struct PeerInfo
 	PeerStatus status;
 	sockaddr_in address;
 	bool bReturnJoinPacket;
```

Comment:
bReturnJoinPacket is set when PeerInfo was cleared in source code. I didn't know if this was intentional. I would like to set it to false when clearing, but didn't look at the source code close enough to make sure it wouldn't negatively affect logic.

Reply:
Yeah, I don't know. It sure looks intentional.